### ISSUE TITLE
Openssl v3 vulnerability in container

### ISSUE THREAD

**Issue by WesSec** [linuxserver/docker-plex#344]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior
The package should be updated as a patch is made available
https://blog.qualys.com/vulnerabilities-threat-research/2022/10/31/qualys-research-alert-prepare-for-a-critical-vulnerability-in-openssl-3-0, versions 3.0 through 3.0.6 are vulnerable

## Current Behavior
A vulnerable version of openssl is used within this plex container

## Steps to Reproduce

1. `docker exec -it plex openssl version`
result : `OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)`

## Environment
**OS:** Ubuntu Server
**CPU architecture:** x86_64
**How docker service was installed:**

## Command used to create docker container (run/create/compose/screenshot)
Docker compose, latest image

**Comment by Roxedus** [comment]
There is [no patch](https://packages.ubuntu.com/jammy/openssl) pushed for Ubuntu Jammy yet.

Also read through this. https://www.linuxserver.io/blog/image-vulnerability-scanning-and-you

**Comment by WesSec** [comment]
```
DON'T

Blindly run vulnerability scanning tools and dump the output into a Github Issue.
```

Hehe im sorry, i did not find this link, maybe add it to the issue template.

**Comment by Roxedus** [comment]
Fixed with 6311d9cc5a26e5f6351a416b606e63592c339655
